scripts/install-cni.sh

```diff
@@ -25,10 +25,25 @@ if [ -z "${pod_cidr:-}" ]; then
 fi
 
 if [ -w /host/etc/cni/net.d ]; then
-  cni_spec=$(echo ${CNI_SPEC_TEMPLATE:-} | sed -e "s#podCidr#${pod_cidr:-}#g")
```

Comment:
How about

1) figure out the values of both pid_cidr and node_ipv6_add;
2) generate the CNI spec once;
?

Reply:
Also, please add logging properly to make debugging easier :-)

Reply:
I'd actually like to add the IPV4 subnet range separately from the IPV6 one, since the IPV6 one is optional, and the entire script should fail if we do not get the IPV4 range.